src/index.js

```diff
@@ -54,6 +56,11 @@ const renderTasks = (tasks) => {
       dots.style.display = 'flex';
       trash.style.display = 'none';
     });
+
+    taskCheckox.addEventListener('change', (e) => {
+      check(e.target, tasks[i]);
+      saveStorage(tasks);
+    });
```

Comment:
For some reason, our data is not properly saved by the localStorage.

Checked item after reload tends to change to the unchecked item.